geoh5py/workspace/workspace.py

```diff
@@ -635,37 +504,64 @@ def fetch_property_groups(self, uid: uuid.UUID) -> List[PropertyGroup]:
 
         return property_groups
 
-    def activate(self):
-        """ Makes this workspace the active one.
+    def fetch_values(self, uid: uuid.UUID) -> Optional[float]:
+        """
+        Fetch the data values from the source h5file
 
-            In case the workspace gets deleted, Workspace.active() safely returns None.
+        :param uid: Unique identifier of target data object
+
+        :return value: Array of values
```

Comment:
missing period and return format
```suggestion
        :param uid: Unique identifier of target data object.

        :return: Array of values.
```